### ISSUE TITLE
How can I change learning rate after "resume_from" a pre-trained model?

### ISSUE THREAD

**Issue by callzhang** [open-mmlab/mmsegmentation#382]
I want to change learning rate schedule after "resume_from" a model. How can I achieve that?
